src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -204,6 +205,43 @@ def manure_spread(aeu):
     return n_list, p_list
 
 
+def ls_factors(lu_strms, areas, avg_slope):
+    results = [0.0] * len(lu_strms)
+    for i in range(len(lu_strms)):
+        if 0 <= avg_slope <= 1.0:
+            m = 0.2
```

Comment:
We can move the calculation of `m` outside the `for`, since it is the same for each iteration